src/components/AppFlow.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useApplicationsInfo } from '../hooks/useApplicationsInfo';
+import { LoadingBox } from '../shared/components/status-box/StatusBox';
+
+const AppFlow: React.FC = () => {
+  const history = useHistory();
+  const { loaded, appExists } = useApplicationsInfo();
+
+  React.useEffect(() => {
+    if (loaded && appExists) {
+      history.push('/applications');
```

Comment:
Do we need to redirect even for applications? Maybe we can just render the application list view component and only redirect if apps do not exist.

Reply:
updated it